frontend/src/metabase/styled-components/theme/spacing.js

```diff
@@ -0,0 +1,4 @@
+export const spacing = {
+  1: "8px",
+  2: "16px",
+};
```

Comment:
Love it! Should we maybe mirror [these options](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/app.js#L68)? Not sure if they're used, but it makes me think we're using `4px` somewhere as well

Reply:
Yes 👍 Changing this.

Reply:
I ended up making a function here...to be consistent with, say, `color()`.

So it's called like this: `space(1)`.

#### Side-thought:

We could develop a library not unlike Tailwind... `marginLeft(1)` could populate the styled component with `margin-left: 8px;`, `margin(2)` could yield `margin: 16px;`, we could have `marginHorizontal(…)`, `marginVertical(…)`  and so on...having used Tailwind, I'd say these full names would be a lot more parseable by humans than `ml-1`, `pt-4` and the rest.